project/Dependencies.scala

```diff
@@ -7,12 +7,12 @@ object Dependencies {
     val catsCore       = "2.6.1"
     val effect         = "3.2.2"
     val fs2            = "3.1.0"
-    val http4s         = "1.0.0-M21"
+    val http4s         = "1.0.0-M24"
```

Comment:
I didn't realise this get updated to `1.0.x`, this should definiately get downgraded to the more stable `0.23.x`

Reply:
Done in 7125667